terraform/services.tf

```diff
@@ -0,0 +1,8 @@
+resource "google_project_service" "gcp_services" {
+  count   = length(var.gcp_services)
+  project = var.gcloud_project
+  service = var.gcp_services[count.index]
+  # (Optional) If true, services that are enabled and which depend on this service should also be disabled when this service is destroyed.
+  # If false or unset, an error will be generated if any enabled services depend on this service when destroying it.
+  disable_dependent_services = true
```

Comment:
I think it's more appropriate to not set `disable_dependent_services`. Rather than disabling services, I want Terraform to fail because it likely signals that we've made a mistake.

Reply:
Agreed, that's a better way of doing it. I changed the code accordingly.